src/Native/Runtime/gcenv.h

```diff
@@ -264,13 +267,9 @@ enum ThreadType
 #define ClrFlsSetThreadType(threadType) SetGCSpecialThread(threadType)
```

Comment:
You can delete `ClrFlsSetThreadType`, `SetGCSpecialThread` and `ThreadType_GC`. They are no longer needed because of previous changes.

Reply:
Also, you should be able to delete SyncBlockCache that you have commented on gitter. It is also dead code.